src/pages/blog/starlingx-harbor.md

```diff
@@ -0,0 +1,255 @@
+---
+templateKey: blog-post
+title: Enhancing StarlingX through Harbor as a Secure Container Registry Solution
+author: Jagatguru Mishra
+date: 2024-09-30T01:32:05.627Z
+
+category: 
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+Learn about using Harbor as a container registry in StarlingX 10.0. Harbor enhances StarlingX’s capabilities by offering robust, secure management of container images and cloud-native artifacts. <!-- more -->
+
+# About Harbor
+[Harbor](https://goharbor.io/ ) is an open-source container registry that enhances security by implementing policies and role-based access control. It ensures images are vulnerability-free through scanning and provides image signing for trust verification.
+
+With the 2.0 release, Harbor becomes the first OCI (Open Container Initiative)-compliant open-source registry capable of storing a wide range of cloud-native artifacts, including container images, Helm charts, OPAs, and Singularity, among others. Helm charts can now be pushed directly to Harbor using Helm3, eliminating the need for separate hosting in ChartMuseum. Instead, they are stored alongside container images under artifacts. As shown in the figure below, container images, Helm charts, and Cloud Native Application Bundles (CNAB) are hosted within the same project.
+
+Further details can be found [here](https://docs.starlingx.io/admintasks/kubernetes/harbor-as-system-app-1d1e3ec59823.html)
+
+
+# What can you do with Harbor
+
+As an administrator, use Harbor if you want to streamline your artifact management, enhance security, and maintain compliance in your cloud-native environment. Here’s how Harbor can help you achieve these goals:
+
+* __Store a Wide Range of Artifacts:__
+With Harbor 2.0, you can store not just container images, but also other OCI (Open Container Initiative)-compliant artifacts like Helm charts, OPAs, and Singularity images. This unified storage simplifies the management of multiple artifact types, allowing users to access and manage all their artifacts in one place. Helm charts, previously hosted separately via ChartMuseum, can now be pushed directly into Harbor using Helm3.

```

Comment:
```suggestion
This unified storage simplifies the management of multiple artifact types, allowing users to access and manage all their artifacts in one place.

```

Reply:
The intro paragraph of Harbor already lists the different artifact types, and mentioned the Helm3 update.